outbox-core/src/test/java/io/praesidio/outbox/stubs/MessageNotFoundException.java

```diff
@@ -0,0 +1,9 @@
+package io.praesidio.outbox.stubs;
+
+import io.praesidio.outbox.values.MessageId;
+
+public class MessageNotFoundException extends RuntimeException {
+    public MessageNotFoundException(MessageId messageId) {
```

Comment:
Can we hide constructor - WDYT?

Reply:
I don't think we can - public `MessageRepository` could use this.

Reply:
Yeah, you are right :)